lib/vmpooler/api.rb

```diff
@@ -258,8 +258,10 @@ def mean(list)
             halt 400, 'Invalid "to" date format, must match YYYY-MM-DD'
           end
 
-          # We do need a from date, however.
-          if from_param.nil? or !validate_date_str(from_param.to_s)
+          # It is also ok if from_param is nil, we will assume this also means 'today'
```

Comment:
I would assume that the `from` param always has to be some date in the past, and that the `to` param could be some date in the past _or_ 'today', if it isn't specified. We already assume that an empty `to` param is 'today' on lines 255-256.

What is the benefit of assuming that the `from` date is 'today' if the param isn't specified?